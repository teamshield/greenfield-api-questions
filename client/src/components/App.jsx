/* eslint no-shadow: "off" */
import React, { Component } from 'react';
import { MuiThemeProvider } from '@material-ui/core/styles';

const AppBarStyled = styled(AppBar)`
  padding: 1.5rem;
`;

class App extends Component {
  render() {
    return (
      <React.Fragment>
        <h4>Questions API</h4>
      </React.Fragment>
    );
  }
}

export default App;
