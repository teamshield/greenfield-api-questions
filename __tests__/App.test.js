import React from 'react';
import renderer from 'react-test-renderer';
import App from '../client/src/components/App';

test('It renders App', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
