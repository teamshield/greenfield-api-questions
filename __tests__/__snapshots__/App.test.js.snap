// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders App 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <header>
    <header
      className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionSticky sc-kgoBCf jRbFr MuiAppBar-colorPrimary"
    >
      Team Uru
    </header>
  </header>
  <main />
</div>
`;
